### ISSUE TITLE
[Functionality] Select what value can be copied to clipboard

### ISSUE THREAD

**Issue by princeteck** [VictorGrycuk/streamdeck-color-picker#15]
I use the plugin on regular bases and found that I am mostly in need to have both then `Name of Color` and `Hex Value` to be copied on my clipboard. currently thing which is copied is `Value to show`. this focus me to also use power tool color picker with your plugin which lengthy process.

we can a drop down where user can select what can be copied to clipboard!

**Comment by VictorGrycuk** [comment]
Hi @princeteck, I will have it in mind for any future update.
However, I can't promise you it will be any time soon.

**Comment by Export33** [comment]
Yes that would be nice. I need to show the color and copy the hex value.
